src/graphql/profile_details.graphql

```diff
@@ -0,0 +1,5 @@
+query ProfileDetails($dtag: String, $utc: timestamp) {
+  stakingParams: staking_params(limit: 1) {
+    params
+  }
+}
```

Comment:
I don't know what this is used for yet

Reply:
Hmmm actually I think maybe we don't need this part.